schedule/yast/releasenotes_origin+unregistered_pvm.yaml

```diff
@@ -0,0 +1,25 @@
+---
+name: releasenotes_origin+unregistered@pvm
```

Comment:
Name should not include platform.  Delete '@pvm'

Reply:
done.